### DISCUSSION TITLE
Facing an issue while deploying swell origin storefront theme.

### DISCUSSION THREAD

**Discussion by charity1475** [swellstores/community#229] [Help] [answered]
I forked and cloned origin theme repo.
I created a swell store front and auth keys.
I managed to start it locally.
But I'm getting an error.
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/58939045/233785872-36de2614-2970-4f17-9783-294916258c44.png">

Also when I visit my store front I get a 404 while the store front exist and has products.
<img width="1275" alt="image" src="https://user-images.githubusercontent.com/58939045/233786221-2d5dc8f2-346b-484f-9791-bbce34718685.png">
Also in the local instance when I try to search  I kind like see products from swell store
<img width="1248" alt="image" src="https://user-images.githubusercontent.com/58939045/233786273-aefdda81-b173-4c70-8868-ade556a48ec9.png">

**Comment by Krislunde** [comment]
I have the same issue. Swell seems have removed storefront integration with the dashboard recently, and a ton of things are broken becaue of it. In the origin storefront, translation strings are broken, and the theme errors out on the root page because it can't find "pages" in the dashboard/api. 

I've reached out to Swell, and while you can still create custom fields and models, there is no way to make them appear in the dashboard or create new entries of these content types, except via CLI/API.

It makes it incredibly frustrating to try to learn using Swell as a developer, at the moment. And hopeless for clients, as even though Swell's website and API touts its capabilities "as a CMS" it's currently unusable as one.

**Reply by charity1475** [reply]
Thank you very much for your response. I decided to user the swell GraphQL API and implement a custom logic.

**Reply by ericingram** [reply]
Thanks for the feedback @Krislunde. We're working to fix the issues that arose from the storefront changes (updated CLI coming soon), by allowing a developer using the CLI to configure and use the theme editor. The challenge is that it's a very complex component and we want to limit exposure to new users until we can assign more resources to it.

There's also a roadmap effort to bring collection models into the dashboard as you described.

**Comment by logeshswell** [comment, marked as answer]
@charity1475 Page translations are fixed in your storefront now. Secondly, Swell has sunsetted the hosted storefront solution. You may choose any hosting provider which is best suited to your needs.

**Reply by logeshswell** [reply]
Screenshots for reference:
<img width="1511" alt="Screenshot 2023-04-25 at 4 37 31 AM" src="https://user-images.githubusercontent.com/100766131/234134760-7eefcc8e-cb80-4938-9312-bfaadbb6e4a0.png">
<img width="1512" alt="Screenshot 2023-04-25 at 4 37 42 AM" src="https://user-images.githubusercontent.com/100766131/234134765-e71d1387-f576-4a55-a8d7-418b07bd9c9b.png">

**Reply by Krislunde** [reply]
@logeshswell Any chance you can fix translations on `dog-street` as well?

**Reply by logeshswell** [reply]
@Krislunde I see that you use a custom storefront. Can you share a screenshot of the translation error that you see, so that I can assist you accordingly.

**Reply by Krislunde** [reply]
@logeshswell It's based on the Origin storefront, so all the translation strings in the Origin storefront would be nice.

**Reply by logeshswell** [reply]
@Krislunde Understood, I updated all the translation strings to your storefront.

**Reply by Krislunde** [reply]
@logeshswell That didn't seem to work. Origin is still throwing translation errors [as seen here](https://imgur.com/a/c8Hy06P), and now the hosted checkout is also throwing (different) translation errors, [as seen here](https://imgur.com/a/Fx6LSED).

Please advice.

**Reply by logeshswell** [reply]
@Krislunde When I navigate to a sample swell-hosted [checkout URL](https://dog-street.swell.store/checkout/02221c854244a809cfa6bd84a50960d4), I see all the translations are showing correctly.
Do you still find this issue? I will also check with the team internally regarding the warning console messages that you shared.

**Reply by logeshswell** [reply]
@Krislunde I fixed your storefront translation issues now. Please check and confirm.
On a side note, make sure that you use the Public key of **Source** type (Storefront) i.e `pk_****OXm`

**Reply by logeshswell** [reply]
<img width="1511" alt="Screenshot 2023-05-12 at 2 51 17 PM" src="https://github.com/swellstores/community/assets/100766131/3b5e9772-b078-46a0-b82f-098ec21125c3">


<img width="1512" alt="Screenshot 2023-05-12 at 2 50 59 PM" src="https://github.com/swellstores/community/assets/100766131/3aef2067-33fd-41ba-ae8e-90976e40eb27">

**Comment by charity1475** [comment]
thanks
